spec/controllers/questions_controller_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+RSpec.describe QuestionsController, :type => :controller do
+  describe "Get #index" do
+    before {get :index}
```

Comment:
Remember the rspec lecture - Given, When, Then. 
`before` is the Given, `it` is the When and Then. 
`get :index` is the When.